src/platform/windows/display_amd.cpp

```diff
@@ -0,0 +1,202 @@
+/**
+ * @file src/platform/windows/display_amd.cpp
+ * @brief Display capture implementation using AMD Direct Capture
+ */
+
+extern "C" {
+#include <libavcodec/avcodec.h>
+#include <libavutil/hwcontext_d3d11va.h>
+}
+
+#include "display.h"
+#include "misc.h"
+#include "src/config.h"
+#include "src/main.h"
+#include "src/video.h"
+
+#include <AMF/components/DisplayCapture.h>
+#include <AMF/components/VideoConverter.h>
+#include <AMF/core/Trace.h>
+
+namespace platf {
+  using namespace std::literals;
+}
+
+namespace platf::dxgi {
+  amd_capture_t::amd_capture_t() {
+  }
+
+  amd_capture_t::~amd_capture_t() {
+    captureComp->Flush();
+    captureComp->Terminate();
+    context->Terminate();
+    // capturedSurface = nullptr;
+    // free(context);
+    // free(captureComp);
+    // capturedSurface.Release();
+    // free(capturedSurface);
+  }
+
+  capture_e
+  amd_capture_t::release_frame() {
+    // BOOST_LOG(error) << "### R.G. release_frame, null pointer?: " << (capturedSurface == nullptr);
+    // if (capturedSurface != nullptr)
+    // {
+    //   BOOST_LOG(error) << "### R.G. Releasing frame";
+    //   Sleep(10);
+    //   capturedSurface->Release();
+    // }
+
+    return capture_e::ok;
+  }
+
+   /**
+   * @brief Get the next frame from the producer thread.
+   * If not available, the capture thread blocks until one is, or the wait times out.
+   * @param timeout how long to wait for the next frame
+   * @param out a texture containing the frame just captured
+   * @param out_time the timestamp of the frame just captured
+   */
```

Comment:
```suggestion
  /**
   * @brief Get the next frame from the producer thread.
   * If not available, the capture thread blocks until one is, or the wait times out.
   * @param timeout how long to wait for the next frame
   * @param out a texture containing the frame just captured
   * @param out_time the timestamp of the frame just captured
   */
```

This should also probably be moved to the header (doxygen can complain if it exists in the header and implementation if there is any mismatch).

Reply:
Not sure what you mean here. I fixed the misaligned comment.

Reply:
I mean to move the documentation block to the `.h` file instead of the `.cpp` file.